lib/index.js

```diff
@@ -1,5 +1,9 @@
-export default (...args) => (req, res, next) => {
+export default (fallback, ...args) => (req, res, next) => {
   if (req.method === 'GET' && req.accepts('html')) {
-    res.sendFile(...args, err => err && next())
+    if (fallback instanceof Function) {
+      fallback(req, res, next)
+    } else {
+      res.sendFile(fallback, ...args, err => err && next())
```

Comment:
That's a [false positive](https://babeljs.io/repl/#?evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&experimental=true&loose=false&spec=false&code=function%20oneOrMoreArguments%20(first%2C%20...rest)%20%7B%0A%20%20console.log(arguments.length%2C%20'arguments')%0A%20%20passThrough(first%2C%20...rest)%0A%7D%0A%0Afunction%20manyArguments%20(...rest)%20%7B%0A%20%20console.log(arguments.length%2C%20'arguments')%0A%20%20passThrough(...rest)%0A%7D%0A%0Afunction%20passThrough%20(foo%2C%20bar)%20%7B%0A%20%20howManyPassedThrough(foo%2C%20bar)%0A%7D%0A%0Afunction%20howManyPassedThrough%20(foo%2C%20bar)%20%7B%0A%20%20console.log(arguments.length%2C%20'arguments'%2C%20typeof%20foo%2C%20typeof%20bar)%0A%7D%0A%0A%2F%2F%20oneOrMoreArguments('one'%2C%20'two'%2C%20'three')%0AoneOrMoreArguments()%0A%0A%2F%2F%20manyArguments('one'%2C%20'two'%2C%20'three')%0AmanyArguments()%0A). 
Arrow functions doesn't even have its own arguments object.

It's also much easier for browser engines to optimize this:
```javascript
(first, ...args) => {
if(first !== undefined) [...]
}
```
than
```javascript
(...args) => {
if(args.length <= 0 && args[0] !== undefined) [...]
}
```
😃